### PR TITLE
Add the logic to install sources

### PR DESCRIPTION
**Pull request by houshengbo**
Fixes: #298

### CODE REVIEW

File: pkg/reconciler/knativeeventing/source/source_test.go [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+	"os"
+	"testing"
+
+	mf "github.com/manifestival/manifestival"
+	eventingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
+	"knative.dev/operator/pkg/reconciler/common"
+	util "knative.dev/operator/pkg/reconciler/common/testing"
+)
+
+func TestAppendInstalledSources(t *testing.T) {
```
Comment (maximilien) [inline pkg/reconciler/knativeeventing/source/source_test.go:31]: Should you do a test with the extreme case? No sources enabled... just to test boundary conditions.

**Review by maximilien — COMMENTED** [review]
Left one comment

**Review by maximilien — APPROVED** [review]
/ok-to-test
/approve
/lgtm
